provision/roles/setup_jenkins/templates/jenkins-pv.yaml.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: jenkins
+  namespace: cccp
+spec:
+  capacity:
+    storage: 5Gi
```

Comment:
This is okay to reference 5Gi here. But we can also keep this back into inventory file which becomes a references for what resources we are expecting from CICO

@dharmit @mohammedzee1000 WDYT?

Reply:
Yes we can parameterize it in some way so that if needed, we can use same Ansible playbooks for pre-install by just changing the values depending on the environment we're deploying it on.